cpp/src/traversal/sssp.cu

```diff
@@ -15,7 +15,7 @@
  */
 
 #include <cugraph/algorithms.hpp>
-#include <cugraph/graph_view.hpp>
+#include <cugraph/experimental/graph_view.hpp>
```

Comment:
We moved graph_view & sssp out from the experimental namespace (actually, we no longer have the experimental namespace).